try.html

```diff
@@ -172,6 +172,10 @@ <h3 id="build"> Build </h3>
     <td><img src='/coverity/scan/3997.svg' alt=''/></td>
     <td><code>https://img.shields.io/coverity/scan/3997.svg</code></td>
   </tr>
+  <tr><th> Coverity Code Advisor On Demand: </th>
+    <td><img src='/coverity/ondemand/streams/1234.svg' alt=''/></td>
+    <td><code>https://img.shields.io/coverity/ondemand/streams/1234.svg</code></td>
```

Comment:
This being informational, we can probably put `<code>https://img.shields.io/coverity/ondemand/streams/STREAM.svg</code>`.

We can also add another custom badge to show the same thing for jobs.